jfrog-applications/jfrog-cli/cli-command-summaries.md

```diff
@@ -0,0 +1,150 @@
+# CLI Command Summaries
+
+## Overview
+
+The command summaries feature enables the recording of JFrog CLI command outputs into the local file system.
+This functionality can be used to generate a summary of the executed CLI command.
+
+Each command execution that incorporates this feature can save data files into the file system.
+These files are then used to create an aggregated summary in Markdown format.
+
+Saving data to the filesystem is essential because CLI command executes in separate contexts.
+Consequently, each command that records new data should also incorporate any existing data into the aggregated markdown.
+This is required because the CLI cannot determine when a command will be the last one executed in a sequence of commands.
+
+
+An instance of how Command summaries are utilized can be observed in the [setup-cli GitHub action](https://github.com/jfrog/setup-jfrog-cli/blob/master/README.md#github-job-summaries).
+This action employs the compiled markdown to generate a comprehensive summary of the entire workflow.
+
+### Currently supported commands:
+`jf rt upload`
+
+![rt-upload-summary-example](../.gitbook/assets/rt-upload-summary.png)
+
+
+`jf rt build-publish`
+
+![rt-upload-summary-example](../.gitbook/assets/build-info-summary.png)
+
+
+`jf scan `
+
+![jf-scan-example](../.gitbook/assets/jf-scan-summary.png)
+
+
+`jf build-scan `
+
+![jf-scan-example](../.gitbook/assets/jf-build-scan-summary.png)
+
+## Notes for Developers 🖥️👩🏻‍💻⚙️
+
+To use the command summaries, you'll need to set the `JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR` environment variable.
+This variable designates the directory where the data files and markdown files will be stored.
+
+###  ⚠️ Attention: Files Remain After CLI Execution
+The CLI does not automatically remove the files as they are designed to remain beyond a single execution.
+As a result, it is your responsibility to you to manage your pipelines and delete files as necessary.
+You can clear the entire directory
+of `JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR` that you have configured to activate this feature.
+
+## How to Implement? 👩‍💻
```

Comment:
The "How to Implement?" section should be a sub-section under the "Notes for Developers" section, meaning the title should have a smaller heading ("###" instead of "##").